aldryn_newsblog/models.py

```diff
@@ -137,7 +149,7 @@ class Article(TranslatedAutoSlugifyMixin,
     tags = TaggableManager(blank=True)
 
     related = SortedManyToManyField('self', verbose_name=_('related articles'),
-                                    blank=True)
+                                    blank=True, symmetrical=ARTICLE_RELATED_SYMMETRICAL)
```

Comment:
aahh... ok :) no - this idea is already abandoned. I know I know. No worries - the migration "set" (let's call it) has to be consistent. We're now discussing not compatible change in addons/CMS channel.